rebar.config

```diff
@@ -1,23 +1,25 @@
-{require_otp_vsn, "R16|17|18|19|20|21"}.
+{require_otp_vsn, "21"}.
 
-{erl_opts, [{parse_transform, lager_transform}, debug_info, warn_untyped_record]}.
+{erl_opts, [debug_info, warn_untyped_record, nowarn_export_all]}.
 
 {eunit_opts, [verbose]}.
 {cover_enabled, true}.
 
 {escript_emu_args, "%%! -escript main cuttlefish_escript +A 0\n"}.
-{escript_incl_apps, [goldrush, getopt, lager]}.
+{escript_incl_apps, [getopt]}.
 
-{xref_checks, []}.
-{xref_queries, [{"(XC - UC) || (XU - X - B - \"(rebar.*|mustache)\" : Mod)", []}]}.
+{deps, [{getopt, {git, "https://github.com/emqx/getopt.git", {tag, "v1.0.2"}}}]}.
 
-{deps, [
-    {getopt, ".*", {git, "https://github.com/basho/getopt.git", {tag, "v0.8.2"}}},
-    {lager, ".*", {git, "https://github.com/basho/lager.git", {tag, "3.2.4"}}},
-    {neotoma, ".*", {git, "https://github.com/basho/neotoma.git", {tag, "1.7.4"}}}
-  ]}.
+{profiles,
+    [{test, [{deps, [bbmustache]}]},
+     {escript, [{post_hooks, [{escriptize, "cp _build/escript/bin/cuttlefish ./"}]}]}
+    ]}.
+
+{provider_hooks, [
+    {pre, [{compile, {neotoma, compile}}]}
+]}.
+
+{post_hooks, [{app_compile, "make escript"}]}.
```

Comment:
This is a convoluted way of triggering escriptize and copying. It might be cleaner to remove the escript profile, move the escriptize post hook to default, and trigger escriptize on app_compile.